SunEngine.Commons/Cache/CachePolicy/CustomCachePolicy.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.DependencyInjection;
+using SunEngine.Commons.Cache.CacheModels;
+using SunEngine.Commons.DataBase;
+
+namespace SunEngine.Commons.Cache.CachePolicy
+{
+    public class CustomCachePolicy : ICachePolicy
+    {
+        public bool CanCache(CategoryCached category, int? page = null)
+        {
+            if (category.Parent?.CacheSettings == null)
```

Comment:
Could you please clarify this. Why we check parent and not the category itself? Is it possible that Parent will have Parent too, should we ignore this case(category.Parent.Parent)?